plugins/flatpak/gs-flatpak.c

```diff
@@ -1665,9 +1665,18 @@ gs_flatpak_add_updates_pending (GsFlatpak *self, GsAppList *list,
 
 		update_priority = get_app_update_priority (app);
 
-		main_app = get_real_app_for_update (self, app, cancellable, error);
-		if (main_app == NULL)
-			return FALSE;
+		main_app = get_real_app_for_update (self, app, cancellable, &error_local);
+		/* do not fail if one of the updates cannot be done (otherwise the update
+		 * list will be empty), just inform the user */
+		if (main_app == NULL) {
+			g_autoptr(GsPluginEvent) event = gs_plugin_event_new ();
```

Comment:
It's not entirely clear to me the effect that this will have to the user; will it create a in-app notification? I want to avoid a gibberish/technical error to pop up to the user if possible.

Reply:
Yes. It's an in-app notification. I did it like that because it matches the approach in the `add_updates` function (used for deploying pre-downloaded updates) which is not usually called in our case.
I can remove this event and let it continue silently (maybe with a warning) if that's better UX.